pkg/ibm/resources_container.go

```diff
@@ -177,21 +179,21 @@ func (resources *ResourcesContainer) CollectResourcesFromAPI() error {
 	}
 
 	for _, region := range resources.regions {
-		err = resources.collectRegionalResources(region, apiKey)
+		err = resources.collectRegionalResources(region, apiKey, resources.ResourceGroupID)
 		if err != nil {
 			return err
 		}
 	}
 
-	err = resources.collectGlobalResources(apiKey)
+	err = resources.collectGlobalResources(apiKey, resources.ResourceGroupID)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (resources *ResourcesContainer) collectRegionalResources(region, apiKey string) error {
+func (resources *ResourcesContainer) collectRegionalResources(region, apiKey, resourceGroupID string) error {
```

Comment:
```suggestion
func (resources *ResourcesContainer) collectRegionalResources(region, apiKey string) error {
```
No need to pass `resourceGroupId` as a parameter. Use `resources.ResourceGroupID` instead.